JitsiConference.js

```diff
@@ -381,11 +381,20 @@ JitsiConference.prototype._init = function(options = {}) {
 
     // Get the codec preference settings from config.js.
     const codecSettings = {
-        jvbDisabledCodec: _getCodecMimeType(config.videoQuality?.disabledCodec),
-        p2pDisabledCodec: _getCodecMimeType(config.p2p?.disabledCodec),
-        enforcePreferredCodec: config.videoQuality?.enforcePreferredCodec,
-        jvbPreferredCodec: _getCodecMimeType(config.videoQuality?.preferredCodec),
-        p2pPreferredCodec: _getCodecMimeType(config.p2p?.preferredCodec)
+        jvb: {
```

Comment:
Question: wouldn't it make sense to compute the new setting here even if the deprecated ones are used and then only use the new one internally?

Reply:
I plan to remove handling of deprecated settings after few releases so its going to look much cleaner then. I don't think its good idea to move all the validation logic here, IMHO it needs to be handled in the CodecSelection constructor itself.